src/js/client/consent.ts

```diff
@@ -58,6 +58,14 @@ export class ConsentForm {
 		);
 	}
 
+	protected getCheckboxes(checked?: boolean): Array<HTMLInputElement> {
+		return Array.from(
+			this.element.querySelectorAll(
+				`.consent-form__section input[type=checkbox]${checked ? ':checked' : ''}`
+			)
+		);
+	}
```

Comment:
`checked` param is not longer needed. Can you please remove it like:

```suggestion
	protected getCheckboxes(): Array<HTMLInputElement> {
		return Array.from(
			this.element.querySelectorAll(
				`.consent-form__section input[type=checkbox]`
			)
		);
	}
```